authenticator/authenticator.go

```diff
@@ -212,9 +215,12 @@ func (a *Authenticator) getAuthRequest() (rest.Request, error) {
 		Nonce:          nonce,
 		Label:          fmt.Sprintf("%s-%d", labelPrefix, labelPostFix),
 		ReadOnly:       a.ReadOnly,
-		ExpirationTime: tokenExpiration,
+		ExpirationTime: a.TokenExpiration,
 		GlobalKey:      !a.Whitelisted,
 	}
+	if authRequest.ExpirationTime == "" {
```

Comment:
This would also be moved to the getter method. Don't forget to test this logic in the `authenticator_test.go`

Reply:
Done